std/datetime.d

```diff
@@ -17605,6 +17599,12 @@ assert(Interval!Date(Date(1996, 1, 2), dur!"years"(3)) ==
             throw new DateTimeException("Arguments would result in an invalid Interval.");
     }
 
+    ///
+    unittest
+    {
+        assert(Interval!Date(Date(1996, 1, 2), dur!"weeks"(3)) ==
+               Interval!Date(Date(1996, 1, 2), Date(1996, 1, 23)));
```

Comment:
Yet another reason *all* code examples need to be unittests rather than unmaintained code in ddoc comments.